apps/gp2-server/test/mocks/squidex-graphql-client-with-server.mock.ts

```diff
@@ -28,10 +29,12 @@ export const getSquidexGraphqlClientMockServer = (): SquidexGraphqlClient => {
     Int: () => 8,
     Instant: () => '2021-10-12T15:42:05Z',
     JsonScalar: () => {},
-    Users: () => getGraphQLUser(),
-    UsersResultDto: resultDto,
     Dashboard: () => getSquidexGraphqlDashboard(),
     DashboardResultDto: resultDto,
+    Users: () => getGraphQLUser(),
+    UsersResultDto: resultDto,
+    WorkingGroups: () => getGraphQLWorkingGroup(),
```

Comment:
inconsistent naming and casing: Here it's getGraphQLWorkingGroup, above it's getSquidexGraphqlDashboard.

QL vs ql
with Squidex in the name or not.

Reply:
well its consistent with Users  (both are getGraphQl .... ) so i guess its a problem we need to solve globally. probably not in this PR?

Reply:
agreed